zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/GetFlowControlActuator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.qa.util.actuator;
+
+import feign.Feign;
+import feign.Headers;
+import feign.RequestLine;
+import feign.Retryer;
+import feign.Target.HardCodedTarget;
+import io.camunda.zeebe.qa.util.cluster.TestApplication;
+import io.zeebe.containers.ZeebeNode;
+
+public interface GetFlowControlActuator {
```

Comment:
:x: this can be removed, right? The `FlowControlActuator` supports both get and set.